roles/lib_openshift/src/test/unit/test_oc_env.py

```diff
@@ -435,6 +451,114 @@ def test_removing_env_vars(self, mock_cmd, mock_tmpfile_copy):
             mock.call(['oc', '-n', 'default', 'get', 'dc', 'router', '-o', 'json'], None),
         ])
 
+    @unittest.skipIf(six.PY3, 'py2 test only')
+    @mock.patch('os.path.exists')
+    @mock.patch('os.environ.get')
+    def test_binary_lookup_fallback(self, mock_env_get, mock_path_exists):
```

Comment:
Ahhh... I think this is the copypasta tests for increasing test coverage? Yeah, I don't like to add more code... more code == harder to maintain. IMHO duplicate tests don't really increase the quality of the project.